Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -38,6 +38,7 @@ def inreplace_pairs(path, replacement_pairs)
       unless contents.errors.empty?
         raise Utils::InreplaceError, path => contents.errors
       end
+      path.atomic_write(contents) if ARGV.include?("--write")
```

Comment:
👍 for documenting this and the other new arguments as well (`--version`, `--mirror`)